src/react-redux/with-api-client.js

```diff
@@ -28,61 +28,82 @@ export default function withApiClient( apiName, options ) {
 			static childContextTypes = WrappedComponent.childContextTypes;
 			static WrappedComponent = WrappedComponent;
 
-			state = { clientKey: null, client: null, clientState: null };
+			constructor( props ) {
+				super( ...arguments );
```

Comment:
is this allowed? seems odd. if using `arguments` I would have expected something like `super.apply( this, arguments )`; if using `...` I would have expected `constructor( ...args ) { super( ...args ); const [ props ] = args;`

maybe it's all fine and I'm just not expecting the right things…

Reply:
Yeah, I picked up this one-line from the Gutenberg source. Seems to work fine there.